src/js/components/CharityPage.jsx

```diff
@@ -136,27 +128,78 @@ const CharityAnalysisSection = ({charity}) => (
 	</div>
 );
 
+//this function gets project data of a certain charity, pushes the most recent projects to the outputarray and returns the total cost
+const getProjectData = (charity, outputarray) => {
+	// Sort all projects by year, descending, and filter out other years
+	let programSplit = charity.projects.sort((a,b) => {
+		if (a.year > b.year) return -1;
+		if (a.year < b.year) return 1;
+		return 0;
+	})
+	let mostRecentYear = programSplit[0].year;
+	programSplit = programSplit.filter(proj =>  proj.year === mostRecentYear)
+	let i = 0;
+
+	programSplit.map((obj) => {
+		const colours = ["#27AE60", "#F2994A", "#2D9CDB", "#C32DDB", "#F24A4F"]
+	
+		if (obj.inputs.findIndex(elem => elem.name === "projectCosts") !== -1) {
+			if (i >= colours.length) i = 0; //limiting colours to only those in the array
+			let insert = {
+				color: colours[i],
+				title: obj.name,
+				value: obj.inputs[obj.inputs.findIndex(elem => elem.name === "projectCosts")].value,
+			};
+			if (insert.value) outputarray.push(insert);
+			else i--;
+			i++;
+		}
+	});
+	const returnObj = {};
+	returnObj.totalProjectValue = outputarray.reduce((acc, elem) => acc + Number(elem.value), 0);
+	returnObj.year = mostRecentYear;
+	return returnObj;
+
+}
+
 const CharityAbout = ({charity}) => {
 	// Safety: in case the url is e.g. wwww.mysite.com with no http(s)
 	let churl = charity.url;
 	if (churl && churl.indexOf('http') !== 0) churl = 'http://'+churl;
+	let pieChartData = [];
+	let totalProjectValue = 0;
+	let mostRecentProjectYear;
+	if (charity.projects !== undefined) {
+		const projectData = getProjectData(charity, pieChartData);
+		totalProjectValue = projectData.totalProjectValue;
+		if (pieChartData.length > 0) {
+			mostRecentProjectYear = ` (${projectData.year})`;
+		}
+	}
+
 	return (
 		<div className='charity-about'>
-			{NGO.getName(charity) !== NGO.displayName(charity)? <h4 className='official-name'>{NGO.getName(charity)}</h4> : null}
+			{/* {NGO.getName(charity) !== NGO.displayName(charity)? <h4 className='official-name'>{NGO.getName(charity)}</h4> : null} */}
 			<CharityAboutImage charity={charity} />
-			<div className='descriptions'>
-				<div className='description-short'>
-					{charity.summaryDescription? <MDText source={charity.summaryDescription} /> : null}
-				</div>
-				<div className='description-long'>
-					{charity.description? <MDText source={charity.description} /> : null}
-				</div>
-			</div>
-			<div className='url'>
-				<a href={churl} target='_blank'>{charity.url}</a>
-			</div>
-			<div className='official-details'>
-				{NGO.registrationNumbers(charity).map(reg => <small key={reg.id}>{reg.regulator}: {reg.id}</small>)}
+			<div className='charity-about-details div-section-text border'>
+				<h3 className='header-section-title'><b>Details on {charity.name}</b></h3>
```

Comment:
Use `NGO.displayName()` instead of `charity.name` -- they're not always the same